src/Symfony/Component/Messenger/Bridge/Kafka/Transport/KafkaReceiver.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Symfony\Component\Messenger\Bridge\Kafka\Transport;
+
+use Psr\Log\LoggerInterface;
+use RdKafka\KafkaConsumer;
+use Symfony\Component\Messenger\Envelope;
+use Symfony\Component\Messenger\Exception\TransportException;
+use Symfony\Component\Messenger\Transport\Receiver\ReceiverInterface;
+use Symfony\Component\Messenger\Transport\Serialization\SerializerInterface;
+
+/**
+ * @author Konstantin Scheumann <konstantin@konstantin.codes>
+ */
+class KafkaReceiver implements ReceiverInterface
+{
+    /** @var LoggerInterface */
+    private $logger;
+
+    /** @var SerializerInterface */
+    private $serializer;
+
+    /** @var RdKafkaFactory */
+    private $rdKafkaFactory;
+
+    /** @var KafkaReceiverProperties */
+    private $properties;
+
+    /** @var KafkaConsumer */
+    private $consumer;
+
+    /** @var bool */
+    private $subscribed;
```

Comment:
```suggestion
    private $subscribed = false;
```

We can remove the initialization from the constructor then.